style.css

```diff
@@ -0,0 +1,108 @@
+body {
+  background-image: url(pics/shapes.png);
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure this `image` is set as `background` only for the `first section` and not of the `body` as you can see on the Figma design.